src/components/contacComponents/contactInfo/ContactInfo.jsx

```diff
@@ -6,7 +7,7 @@ export default function ContactInfo() {
     <div className="container">
       <div className="left-wraper">
         <div className="left">
```

Comment:
The code gets a little bit hard to read with all the containers, wrappers, rights, lefts etc. I sent you an idea on how to clean it up on Discord (which also solves the vertical centering issue)